geps/gep-3171/index.md

```diff
@@ -0,0 +1,174 @@
+# GEP-3171: Percentage-based Request Mirroring
+
+* Issue: [#3171](https://github.com/kubernetes-sigs/gateway-api/issues/3171)
+* Status: **Provisional**
```

Comment:
```suggestion
* Status: **Implementable**
```

Reply:
Nice catch, @robscott -- once this change is made I'll happily add an LGTM. 🙂